src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -583,7 +583,7 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 <Target Name="_GetReferenceAssemblyPaths">
 	<GetReferenceAssemblyPaths
 			TargetFrameworkMoniker="$(TargetFrameworkIdentifier),Version=v1.0"
-			RootPath="$(TargetFrameworkRootPath)">
+			RootPath="$(XATargetFrameworkRootPath)">
```

Comment:
> Where is the value for `$(XATargetFrameworkRootPath)` defined? 

Where is the value for `$(TargetFrameworkRootPath)` defined? It isn't, not in normal customer projects. So *normally* this passes an empty string, which causes `<GetReferenceAssemblyPaths/>` to search its global directories.

Overriding `$(TargetFrameworkRootPath)` is documented in `UsingJenkinsBuildArtifacts.md`, and exists so that our "side-loaded install" is used instead of any existing system-wide install. The problem is that if you actually do override `$(TargetFrameworkRootPath)`, it impacts *everything*, as mentioned in the commit message.

Primary use remains the same: by default `$(XATargetFrameworkRootPath)` will be unset, causing `<GetReferenceAssemblyPaths/>` to search global directories. By using a different property name, we don't clobber/destroy PCL lookup.